src/index.html

```diff
@@ -17,16 +17,44 @@
     />
```

Comment:
The link to the stylesheet is correct, but make sure the path is accurate relative to the HTML file location.